### ISSUE TITLE
Quantum 

### ISSUE THREAD

**Issue by jpcavanagh** [keesverruijt/mayara#2]
Let me know if I can help with Raymarine quantum. I have opencpn radar working with quantum.

**Comment by keesverruijt** [comment]
That will be very useful, JP! I'm not quite ready to add Quantum onto Mayara but this will come soon.

**Comment by LasseGH22** [comment]
I misread the part of the README saying that Quantum is planned as supported. But i was able to get your project to identify my Raymarine Quantum Q24C radar automatically. It not being supported also makes it make sense that whenever i tried to change anything in the settings available in the webinterface it just said "shutdown"

**Comment by keesverruijt** [comment]
Quantum support has been improved in November-December, can you guys test this?

**Comment by jpcavanagh** [comment]
Yes I will but am at sea right now so not before end of next week

John

**Comment by keesverruijt** [comment]
Quantum works.
